.docker/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM php:8-fpm-alpine as runtime
+
+RUN apk update \
+  && apk add --no-cache curl make coreutils icu-libs icu-dev libzip-dev libxslt-dev php-mysqli php-pdo_mysql postgresql-dev libpng-dev
+
+RUN NPROC=$(grep -c ^processor /proc/cpuinfo 2>/dev/null || 1) \
+    && docker-php-ext-install -j${NPROC} zip intl xsl mysqli pdo_mysql pdo_pgsql gd
+
+FROM runtime AS build
```

Comment:
The first FROM is clearly mandatory, but are these "extra FROM" statements necessary?

FROM runtime AS build
...
FROM build AS dev
...

Reply:
Answer: https://github.com/ilstefano/gacela-example/pull/1#commitcomment-73399516
> Not yet. These are instructions to separate build commands into 'stages' (FROM <stage>) which help to easily use the same Dockerfile and `docker build' command during a development lifecycle. As IMHO everybody sooner or later builds her images for integrations or deployments to other docker environments (test, prod) this makes sense to leave it already here.
> TL;DR: are not needed technically without a complexer workflow, but spend semantical added value

Reply:
Thanks for clarifying. I am totally ok with that :)